### ISSUE TITLE
Should we archive this repository?

### ISSUE THREAD

**Issue by HealthyPear** [HealthyPear/protopipe-grid-interface#78]
As reported in cta-observatory/protopipe#201

With the advent of DataPipe and magic-cta-pipe, this project is basically obsolete.

I do not see any further development since I left my previous position.

Should we archive this repository as read-only?

@kosack

**Comment by kosack** [comment]
Yes, I think we can archive this one. It's not really magic-cta-pipe that replaces it, but the new Dirac ProductionSystem and related workflows that nearly replaces this (though still not 100%)

**Comment by HealthyPear** [comment]
People can still use it, the point is that I think no one will further improve this
